integrations/extensions/starter-kits/language-model-conversational-search/README.md

```diff
@@ -72,8 +72,8 @@ Below is a list of the session variables used in this example. Most of them are
 
 - `discovery_date_version` - Discovery date versions are documented in the [release notes](https://cloud.ibm.com/docs/discovery-data?topic=discovery-data-release-notes).
 - `discovery_project_id`: You **MUST** set this value to be the project ID obtained [when configuring Watson Discovery above](#configure-watson-discovery-extension).
-- `model_id`: The id of the watsonx model that you select for this action. Defaults to `google/flan-ul2`.
-- `model_input`: The input to the watsonx model. You MAY change this to do prompt engineering, but a default will be used by the model if you don’t pass a prompt here.
+- `model_id`: The id of the watsonx model that you select for this action. Defaults to `meta-llama/llama-2-70b-chat`.  If you keep this default, be sure to comply with the [Acceptable Use Policy for this model](https://ai.meta.com/llama/use-policy/).
+- `model_input`: The input to the watsonx model. This is set in an expression in Step 5 of the "Generate Answer" action.  You MAY change that expression to do prompt engineering.  If you wish to do so and are using the default model, be sure to research [guidelines for prompting Llama 2](https://www.pinecone.io/learn/llama-2/).  In our experience, this combination of prompt and model is quite effective at producing high-quality answers when it has useful content and it does _often_ say that it doesn't know when it does not have useful content (as instructed in our prompt).  However, it does _sometimes_ provide answers that are not supported by its evidence so consider other models, prompt expressions, or additional logic to reduce the generation of answers that are not
```

Comment:
>However, it does _sometimes_ provide answers that are not supported by its evidence so consider other models, prompt expressions, or additional logic to reduce the generation of answers that are not

That are not....? forgot something to add?

Reply:
Fixed!